pycbc/workflow/grb_utils.py

```diff
@@ -365,3 +400,177 @@ def get_sky_grid_scale(
+    wf.add_node(node)
+    pp_outs.append(trig_files)
+
+    # Trig clustering for each trig file
+    exe_class = select_grb_pp_class(wf, "trig_cluster")
+    job_instance = exe_class(wf.cp, "trig_cluster")
+    for trig_file in trig_files:
+        # Create and add nodes
+        node, out_file = job_instance.create_node(trig_file, pp_dir)
+        wf.add_node(node)
+        pp_outs.append(out_file)
+
+    # Find injections from triggers
+    exe_class = select_grb_pp_class(wf, "inj_finder")
+    job_instance = exe_class(wf.cp, "inj_finder")
+    inj_find_files = FileList([])
+    for inj_tag in inj_tags:
+        tag_inj_files = FileList([f for f in inj_files
+                                  if inj_tag in f.tags])
+        tag_insp_files = FileList([f for f in inj_insp_files
+                                   if inj_tag in f.tags[1]])
+        node, inj_find_file = job_instance.create_node(
+                                           tag_inj_files, tag_insp_files,
+                                           pp_dir)
+        wf.add_node(node)
+        inj_find_files.append(inj_find_file)
+    pp_outs.append(inj_find_files)
+
+    # Combine injections
+    exe_class = select_grb_pp_class(wf, "inj_combiner")
+    job_instance = exe_class(wf.cp, "inj_combiner")
+    inj_comb_files = FileList([])
+    for in_file in inj_find_files:
+        if 'DETECTION' not in in_file.tags:
+            node, inj_comb_file = job_instance.create_node(in_file,
+                                                           pp_dir,
+                                                           in_file.tags,
+                                                           segment)
+            wf.add_node(node)
+            inj_comb_files.append(inj_comb_file)
+    pp_outs.append(inj_comb_files)
+
+    return pp_outs
+
+
+class PycbcGrbTrigCombinerExecutable(Executable):
+    """ The class responsible for creating jobs
+    for ''pycbc_grb_trig_combiner''.
+    """
+
+    current_retention_level = Executable.ALL_TRIGGERS
+
+    def __init__(self, cp, name):
+        super().__init__(cp=cp, name=name)
+        self.trigger_name = cp.get('workflow', 'trigger-name')
+        self.trig_start_time = cp.get('workflow', 'start-time')
+        self.num_trials = int(cp.get('trig_combiner', 'num-trials'))
+
+    def create_node(self, ifo_tag, seg_dir, segment, insp_files,
+                    out_dir, tags=None):
+        node = Node(self)
+        node.add_opt('--verbose')
+        node.add_opt("--ifo-tag", ifo_tag)
+        node.add_opt("--grb-name", self.trigger_name)
+        node.add_opt("--trig-start-time", self.trig_start_time)
+        node.add_opt("--segment-dir", seg_dir)
+        node.add_input_list_opt("--input-files", insp_files)
+        node.add_opt("--user-tag", "PYGRB")
+        node.add_opt("--num-trials", self.num_trials)
+        # Prepare output file tag
+        user_tag = "PYGRB_GRB{}".format(self.trigger_name)
```

Comment:
F-strings are great, use them:
```suggestion
        user_tag = f"PYGRB_GRB{self.trigger_name}"
```